### PR TITLE
Chat and enhancement

### PR DESCRIPTION
**Pull request by mohamed-ahmed-yousef**

### CODE REVIEW

File: apps/web/app/[locale]/(auth)/confirm-otp/page.tsx [hunk 1]
```diff
@@ -117,7 +125,14 @@ export default function ConfirmPinCode() {
...
 								>
-									{resentOtp.isPending ? <Spinner /> : t("resent")}
+									{resentOtp.isPending ? (
+										<>
+											{" "}
+											{t("resent")} &nbsp; <Spinner />{" "}
+										</>
+									) : (
+										t("resent")
+									)}
```
Comment (MuhammadSawalhy) [inline apps/web/app/[locale]/(auth)/confirm-otp/page.tsx:135]: Why not using a prop for the loading state? 😥

File: apps/web/app/[locale]/(auth)/otp-by-email/page.tsx [hunk 2]
```diff
@@ -42,27 +42,31 @@ export default function ResetPassword() {
...
-							disabled={isPending}
-						>
-							{isPending ? <Spinner /> : t("reset-password")}
+						<Button variant="submit" disabled={isPending} className="w-full">
+							{isPending ? (
+								<>
+									{" "}
+									{t("reset-password")} &nbsp; <Spinner />{" "}
+								</>
+							) : (
```
Comment (MuhammadSawalhy) [inline apps/web/app/[locale]/(auth)/otp-by-email/page.tsx:67]: .

**Review by MuhammadSawalhy — CHANGES_REQUESTED** [review]

**Comment by MuhammadSawalhy** [comment]
Please resolve the conflict to be able to merge the PR

File: apps/web/app/[locale]/(auth)/confirm-otp/page.tsx (hunk 1 above)
Comment (mohamed-ahmed-yousef) [reply apps/web/app/[locale]/(auth)/confirm-otp/page.tsx:135]: حاسيت انها مش مستاهله انها تتعمل component

**Pull request merged**